Src/MultiSafepay/MultiSafepay.csproj

```diff
@@ -31,8 +31,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
Reverted Newtonsoft version to not force a major version update (10.x -> 11.x) for clients. As long as there is no .NET core build target, the Newtonsoft version also doesn't have to be .NET core/standard compatible.

Reply:
@richardkooiman Newtonsoft was 11.0.2 any reason why you downgraded to 10.0.3?

Reply:
I tried explaining this above in my first comment. Until there is a .NET core/standard build target, there is no need yet for Newtonsoft 11.x. Since the package reference is a minimum version requirement, 10.x will suffice and 11.x will also work if clients want to use that version. But this way it will not force clients to update to 11.x.
In our specific case we'd like not to be forced to upgrade Newtonsoft to 11.x yet while it's not needed, since all referenced internal packages would also have to be upgraded.

Reply:
@richardkooiman 11.0.2 was implemented while ago as previous versions of Newtonsoft was not Mac compatible - May 29, 2018
https://github.com/MultiSafepay/.Net/commit/863cbcc568c61a813980ddd38bda04905fd05832

Is possible to revert the Newtonsoft version changes, so I can review the AfterPayOrder changes?

Reply:
@iexperto The MultiSafepay .Net package still isn't Mac compatible, because its target framework is not .NET core or .NET standard, but .NET Framework (4.0). So because the entire MultiSafepay .Net package is not yet cross platform compatible, the Newtonsoft package also doesn't have to be.

Reply:
I'd prefer not to revert the Newtonsoft changes unless it's really needed because all of my current clients' internal packages would be forced to update the Newtonsoft package aswell.

Reply:
@richardkooiman The version released in NuGet at the moment is compatible with .NET core.
We changed Newtonsoft to 11.0.2 because is when they supported also .NET core.
Downgrading, other users using newer versions, might have issues.
In this pull request mainly is the addition of AfterPayOrder, please only apply the changes for AfterPayOrder  to be reviewed.
Then you still want to "downgrade" Newtonsoft  please create another pull request for it.
Please let me know if that's Ok with you

Reply:
@iexperto Done. I let my current client know they will have to update the Newtonsoft package in all their projects aswell. Please let me know if you find anything.

Reply:
@richardkooiman , thank you.